src/services/navigator/helpers.test.ts

```diff
@@ -863,3 +873,55 @@ describe('mergeDocuments', () => {
     });
   });
 });
+
+describe('add route', () => {
+  const doc = parser.parseFromString(navDocStackNavigatorSource);
+
+  it('should find 2 route elements', () => {
+    const navigators = doc.getElementsByTagNameNS(
+      Namespaces.HYPERVIEW,
+      'navigator',
+    );
+    const routes = getChildElements(navigators[0]);
+    expect(routes.length).toEqual(2);
+  });
+
+  describe('add stack route', () => {
```

Comment:
maybe that describe text can be more specific to what the nested tests are about

Reply:
Fixed